src/Classes/Providers/ParcelProvider.ps1

```diff
@@ -320,7 +320,12 @@ class ParcelProvider
         return [string]::Empty
     }
 
-    [string] GetSourceArgument([ParcelPackage]$_package)
+    [array] GetSourceArgument([ParcelPackage]$_package)
+    {
+        return [array]::Empty
+    }
+
+    [string] GetImageArgument([ParcelPackage]$_package)
```

Comment:
If this method is specific to only DISM, you don't need to add it here.

Reply:
I also don't think I'll bother with image as I mentioned earlier.